smartsim/_core/_cli/build.py

```diff
@@ -321,6 +324,18 @@ def execute(
 
 def configure_parser(parser: argparse.ArgumentParser) -> None:
     """Builds the parser for the command"""
+
+    available_devices = []
+    for platform in DEFAULT_MLPACKAGES:
+        if (
+            # (platform.operating_system == OperatingSystem.autodetect()) and
+            # (platform.architecture == Architecture.autodetect())
+            (platform.operating_system == OperatingSystem.LINUX) and
+            (platform.architecture == Architecture.X64)
```

Comment:
Sanity checking here: should this be reverted? I'm guessing this should be comparing to the actual hardware